app/assets/stylesheets/organizations.scss

```diff
@@ -0,0 +1,19 @@
+.org-info {
+  text-align: center;
+  border: 1px solid #FF851B;
+  margin-top: 5%;
+}
+
+.button-row {
+  margin: 5%;
+}
+
+.opp-info {
+  text-align: center;
+  border: 1px solid #FF851B;
```

Comment:
Color literals like `#FF851B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF851B` should be written as `#ff851b`